runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -260,7 +344,10 @@ api.nvim_set_decoration_provider(namespace, {
   end,
 })
 
-return setmetatable(M, {
+--- @overload fun(bufnr: number, enable: boolean) Enable/disable inlay hints for a buffer.
+--- bufnr: Buffer handle, or 0 for current
+--- enable: true/false to enable/disable, nil to toggle
```

Comment:
Unfortunately this only works for LuaLS and doesn't show up in generated Vimdocs

Reply:
Figured something out :+1: